upd/MaprDBUpdater.go

```diff
@@ -0,0 +1,162 @@
+package main
```

Comment:
This file seems to be an "app" by itself.
It should be placed under `cmd/upd/main.go`
The main applications should all be in `cmd` folder.
https://github.com/golang-standards/project-layout

Reply:
You are right, that's a separate app. This project serves several purposes: reserve nodes and update nodes in db. In both cases docker image is built but different files are taken into account.

Reply:
@alaingilbert

Reply:
yes. The standard layout that I have seen would look like:
```
vmrenter/
├── cmd/
│   ├── ConfigLoader
│   │   └── main.go
│   ├── MaprDBUpdater
│   │   └── main.go
│   |   ...
...
```

Reply:
@alaingilbert Done